test/test_vmap.py

```diff
@@ -1455,11 +1455,93 @@ def vector_hessian_product(*vectors):
         self._vmap_test(vector_hessian_product, batched_vectors,
                         check_propagates_grad=False)
 
+    def _test_arithmetic(self, op, device, test_grad_grad=True):
+        x = torch.randn(2, 3, requires_grad=True, device=device)
+        y = torch.rand(2, 3, device=device).clamp_min_(0.1).requires_grad_()
```

Comment:
nit: you could have a `_get_rand_no_zeros()` as you seem to do this quite a lot in this file.

Reply:
Updated with a _get_rand_no_zeros() funciton